src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -206,6 +206,14 @@ public function addRequestData(string $key, $value): void
         $this->request->updateContent([$key => $value]);
     }
 
+    /** @param string|int|bool|array $value */
+    public function replaceRequestData(string $key, $value): void
```

Comment:
I'm not familiar with union types, but can't it be set like so?

```suggestion
    public function replaceRequestData(string $key, string|int|bool|array $value): void
```

Reply:
We can, and we should :p

Reply:
There is second case of this close to current func.